samples/drivers/watchdog/prj.conf

```diff
@@ -1,3 +1,4 @@
 CONFIG_LOG=y
 CONFIG_WDT_LOG_LEVEL_DBG=y
 CONFIG_WATCHDOG=y
+CONFIG_WDT_DISABLE_AT_BOOT=n
```

Comment:
didn't your previous patch made it disabled by default?

Reply:
You mean "boards: set CONFIG_WDT_DISABLE_AT_BOOT=y for all Atmel SAM boards" commit? Yes, the patch disables the watchdog for all the Atmel SAM boards. In case of `samples/drivers/watchdog` application we don't want to do that. I.e. since we are showcasing watchdog usage we need to keep it enabled.

Reply:
Github again messed up patches order then. Nevermind